docs/getting-started-with-scalardb.md

```diff
@@ -36,206 +185,59 @@ You can create a JSON file `emoney.json` with the JSON below.
-                      .build();
-      Optional<Result> result = tx.get(get);
-
-      int balance = -1;
-      if (result.isPresent()) {
-        balance = result.get().getInt(BALANCE);
-      }
-
-      // Commit the transaction
-      tx.commit();
-
-      return balance;
-    } catch (Exception e) {
-      tx.abort();
-      throw e;
-    }
-  }
-
-  public void close() {
-    manager.close();
-  }
-}
-```
-
-Now we can run the application.
-
-- Charge `1000` to `user1`:
+- Credit **1000** to **customer1** by running the following command:
 ```shell
-$ ./gradlew run --args="-action charge -amount 1000 -to user1"
+$ ./gradlew run --args="-action charge -amount 1000 -to customer1"
 ```
 
-- Charge `0` to `merchant1` (Just create an account for `merchant1`):
+- Create an account for **merchant1** that has a balance of **0** by running the following command:
 ```shell
 $ ./gradlew run --args="-action charge -amount 0 -to merchant1"
 ```
 
-- Pay `100` from `user1` to `merchant1`:
+- Charge **100** from **customer1** to **merchant1** by running the following command:
 ```shell
-$ ./gradlew run --args="-action pay -amount 100 -from user1 -to merchant1"
+$ ./gradlew run --args="-action pay -amount 100 -from customer1 -to merchant1"
 ```
 
-- Get the balance of `user1`:
+- Get the balance of **customer1** by running the following command:
 ```shell
-$ ./gradlew run --args="-action getBalance -id user1"
+$ ./gradlew run --args="-action getBalance -id customer1"
 ```
 
-- Get the balance of `merchant1`:
+- Get the balance of **merchant1** by running the following command:
 ```shell
 $ ./gradlew run --args="-action getBalance -id merchant1"
 ```
 
-## Further reading
-
-These are just simple examples of how ScalarDB is used. For more information, please take a look at the following documents.
-
-* [Design Document](design.md)
-* [ScalarDB Sample](https://github.com/scalar-labs/scalardb-samples/tree/main/scalardb-sample)
-* [Java API Guide](api-guide.md)
-* Javadoc
-    * [scalardb](https://javadoc.io/doc/com.scalar-labs/scalardb/latest/index.html) - A library that makes non-ACID distributed databases/storages ACID-compliant
-    * [scalardb-rpc](https://javadoc.io/doc/com.scalar-labs/scalardb-rpc/latest/index.html) - ScalarDB RPC libraries
-    * [scalardb-server](https://javadoc.io/doc/com.scalar-labs/scalardb-server/latest/index.html) - ScalarDB Server that is the gRPC interface of ScalarDB
-* [Requirements in the underlying databases](requirements.md)
-* [Schema Loader](schema-loader.md)
-* [How to Back up and Restore](backup-restore.md)
-* [Multi-storage Transactions](multi-storage-transactions.md)
-* [Two-phase Commit Transactions](two-phase-commit-transactions.md)
-* [ScalarDB Server](scalardb-server.md)
+## Reference
+
+To see the source code for the electronic money application used in this tutorial, see [`ElectronicMoney.java`](./getting-started/src/main/java/sample/ElectronicMoney.java).
```

Comment:
In the original document, we showed the source code of `ElectronicMoney.java`. However, the code shown in the doc was different than the code in the `.java` file.

To avoid needing to update the source code for `ElectronicMoney.java` twice (in the source code and in the doc), I've simply added a link to the source code as a reference.